content/src/content/jcr_root/apps/core/wcm/components/tabs/v1/tabs/tabs.html

```diff
@@ -20,15 +20,18 @@
     <ol data-sly-test="${tabs.items && tabs.items.size > 0}"
         role="tablist"
         class="cmp-tabs__tablist"
-        aria-multiselectable="false">
-        <li data-sly-repeat.item="${tabs.items}"
-            role="tab"
-            class="cmp-tabs__tab${item.name == tabs.activeItem ? ' cmp-tabs__tab--active' : ''}"
-            data-cmp-hook-tabs="tab">${item.title}</li>
+        aria-multiselectable="false"
+        data-sly-list.tab=${tabs.items}>
```

Comment:
`data-sly-list.tab=${tabs.items}` -> `data-sly-list.tab="${tabs.items}"`

Reply:
Solved, thx for finding this 👍